lib/stitches/configuration.rb

```diff
@@ -13,6 +13,8 @@ def reset_to_defaults!
     @custom_http_auth_scheme = UnsetString.new("custom_http_auth_scheme")
     @env_var_to_hold_api_client_primary_key = NonNullString.new("env_var_to_hold_api_client_primary_key","STITCHES_API_CLIENT_ID")
     @env_var_to_hold_api_client= NonNullString.new("env_var_to_hold_api_client","STITCHES_API_CLIENT")
+    @max_cache_ttl = NonNullInteger.new("max_cache_ttl", 5 * 60)
+    @max_cache_size = NonNullInteger.new("max_cache_size", 100)
```

Comment:
I'm worried that arbitrary defaults will cause confusion when people update the gem and don't know that the caching was introduced. I would prefer to remove the defaults entirely, releasing this as a breaking change that forces users to make a decision about (thus know about) caching TTLs.

Reply:
Thanks for taking a look. I'm unclear on your recommendation.

Are you saying: major version bump AND keep the defaults

Or are you saying: major version bump AND remove the defaults

Reply:
My recommendation would be the latter - major version bump, and remove the defaults. My rationale is that by removing the defaults, we ensure people configure a reasonable TTL for their implementation when they upgade the gem. Otherwise they might end up getting this new functionality without realizing it, which could be pretty risky.

Reply:
Got it. I've updated it so that a value of 0 will disable the cache, and I've made that the default.